node/src/components/consensus/protocols/highway/tests.rs

```diff
@@ -151,6 +151,68 @@ fn send_a_wire_unit_with_too_small_a_round_exp() {
     }
 }
 
+#[test]
+fn send_a_valid_wire_unit() {
+    let creator: ValidatorIndex = ValidatorIndex(0);
+    let state: State<ClContext> = new_test_state(&[100.into()], 0);
+    let panorama: ValidatorMap<Observation<ClContext>> = Panorama::from(vec![N, N]);
+    let seq_number = panorama.next_seq_num(&state, creator);
+    let mut rng = TestRng::new();
+    let wunit: WireUnit<ClContext> = WireUnit {
+        panorama,
+        creator,
+        instance_id: ClContext::hash(INSTANCE_ID_DATA),
+        value: Some(CandidateBlock::new(
+            ProtoBlock::new(vec![], vec![], false),
```

Comment:
I think @goral09 just [sabotaged your PR](https://github.com/CasperLabs/casper-node/pull/653). :grin: 
This will not produce `ValidateConsensusValue` anymore since the proto block is empty.

Reply:
I merged the new master, but ValidateConsensusValue is still being produced in the test :thinking: